revvy/robot_config.py

```diff
@@ -72,6 +72,21 @@ def dict_get_first(dictionary, keys):
     return None
 
 
+def make_script_name_common(script_idx, assignment_type, detail):
+    return 'script_{}_{}_{}'.format(
```

Comment:
Why is this not an f-string?

Reply:
Why not?
This is readable and indicates without looking into that string is mostly composed of dynamic arguments.
Visually I split the format string  and the arguments in a separate line, so the line becomes shorter.
If you want, I can convert it to f-string.